src/main/java/uk/gov/justice/hmpps/probationteams/controllers/ProbationAreaController.kt

```diff
@@ -20,6 +21,29 @@ import uk.gov.justice.hmpps.probationteams.services.SetOutcome
         produces = [APPLICATION_JSON_VALUE])
 class ProbationAreaController(val localDeliveryUnitService: LocalDeliveryUnitService) {
 
+    @GetMapping(
+            path = ["/{probationAreaCode}"],
+            produces = [APPLICATION_JSON_VALUE])
+
+    @ApiOperation(value = "Retrieve a Probation Area", nickname = "Retrieve a Probation Area")
+    @ApiResponses(value = [
+        ApiResponse(code = 404, message = "Probation Area not found"),
```

Comment:
guess technically it doesn't return 404 if probation area not found

Reply:
True.  Too much haste / cut'n-paste.